### ISSUE TITLE
[bug]: convert_model does nothing more than generate an image

### ISSUE THREAD

**Issue by VisionaryMind** [invoke-ai/InvokeAI#2734]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

8GB

### What happened?

It is not possible to convert or import models from the CLI.

At first, I presumed it was an issue with the build I was using, but without the ability to perform a `git pull` to update components, I had to re-download and install from scratch to ensure the latest was being used. Unfortunately, that does not fix the problem. Running `import_model` or `convert_model` from the CLI does nothing more than generate an image in the output folder. No converted file is created nor items added to the model registry / list in the UI. Here is what the CLI shows when running both commands:

```
>> Patchmatch initialized
100%|██████████████████████████████████████████████████████████████████████████████████| 50/50 [00:09<00:00,  5.51it/s]
Generating: 100%|████████████████████████████████████████████████████████████████████████| 1/1 [00:09<00:00,  9.38s/it]

>> Usage stats:
>>   1 image(s) generated in 10.34s
>>   Max VRAM used for this generation: 2.81G. Current VRAM utilization: 2.17G
>>   Max VRAM used since script start:  2.81G
Outputs:
[2] D:\Downloads\invokeAI\outputs\000001.1212095919.png: convert_model D:\\Downloads\\models\\model_to_import.safetensors -s 50 -S 1212095919 -W 512 -H 512 -C 7.5 -A k_lms
```
Alternately, if I run `invoke` with the --autoconvert option, pointing to a directory, it sits on `Patchmatch initialized` indefinitely. The most I have waited before killing the process is 30 minutes. I see no evidence that it is doing anything, but if it takes longer than that, please advise.

**Comment by gogurtenjoyer** [comment]
Quick question: are you using `!` before these commands? Without that, they're being treated as prompts, which explains why the end result is a generated image.

**Comment by VisionaryMind** [comment]
@gogurtenjoyer Yes, that was it! Missing the need for a `!`. Thanks, it's working now.

**Comment by VisionaryMind** [comment]
I have closed this issue, but will at least leave a comment here that conversion doesn't work for certain models. For our test, we used [Realistic Vision V1.3](https://civitai.com/api/download/models/6987?type=Model&format=SafeTensor) (which has been optimized for InvokeAI), yet post-conversion, the prompts are not even remotely close. Asking for a man-dear humanoid, for example, produces a photo of a little girl. So back to Automatic1111, unless you have plans to update the conversion algorithm. We'll try it on a couple of other files, but I have read in other threads that conversion doesn't work for many of the models found on CivitAI. It appears to be a common problem.
